core/domain/feedback_services.py

```diff
@@ -366,6 +366,30 @@ def _get_thread_from_model(thread_model):
         thread_model.created_on, thread_model.last_updated)
 
 
+def get_new_messages_for_exp_id(exp_id):
+    """Fetch the feedback threads for the given exploration, and use those to
+    finally get the new (open) feedback messages.
+
+    Args:
+        exp_id: str.
+
+    Returns:
+       A list of FeedbackMessage.
+    """
+    thread_models = (
+        feedback_models.FeedbackThreadModel.get_latest_updated_threads(exp_id))
```

Comment:
I think `get_most_recently_updated_threads()` would be a better name, as per your suggestion in the comment above.